x-pack/solutions/security/plugins/security_solution/common/api/endpoint/actions/response_actions/running_procs/running_procs.schema.yaml

```diff
@@ -22,10 +22,37 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '../../../model/schema/common.schema.yaml#/components/schemas/SuccessResponse'
+                $ref: '#/components/schemas/GetProcessesRouteResponse'
 
 components:
   schemas:
     GetProcessesRouteRequestBody:
+      example:
+        endpoint_ids:
+          - "ed518850-681a-4d60-bb98-e22640cae2a8"
       allOf:
         - $ref: '../../../model/schema/common.schema.yaml#/components/schemas/BaseActionSchema'
+    GetProcessesRouteResponse:
```

Comment:
We should include a note on this one indicating that for SentinelOne, the results are in a zip file that must be downloaded via the `/api/endpoint/action/{action_id}/file/{file_id}/download` api

(Future: we can probably improve this response to include the relative URL to the download in the `outputs`)

Reply:
@natasha-moore-elastic FYI